README.md

```diff
@@ -91,6 +91,25 @@ The webserver comes with some predefined response status codes for certain paths
 | /service-unavailable    | 503           |
 | /gateway-timeout        | 504           |
 
+### Custom HTTP verbs
+
+    $ docker run -d -p 80:80 -e 'allow_http_verbs=FOO,BAR' campanda/webserver-mock
+    // or you can with -it instead to check the logs
+
+By default only `GET` and `POST` are allowed.
```

Comment:
this comment is irritating. "you can only do A and B" next line: "do C" => works - looks strange